config/findbugs-exclude.xml

```diff
@@ -169,4 +169,10 @@
         <Bug pattern="RCN_REDUNDANT_NULLCHECK_WOULD_HAVE_BEEN_A_NPE"/>
     </Match>
 
+    <!-- AWS use UNIX-style line endings -->
+    <Match>
+        <Class name="com.mongodb.internal.connection.AuthorizationHeader"/>
+        <Bug pattern="VA_FORMAT_STRING_USES_NEWLINE"/>
+    </Match>
```

Comment:
Is this necessary for the tests to pass on Windows? I suspect the change made in `AuthorizationHeader.java:118` fixes the issue since it no longer uses platform-specific newline characters.